log_api.go

```diff
@@ -0,0 +1,156 @@
+package lorg
+
+import (
+	"io"
+	"os"
+)
+
+const (
+	// DefaultFormatting describes default format for log records
+	// If you don't run SetFormat, Log instance will use Format instance with
+	// this given formatting.
+	//
+	// See Format structure documentation for information about `${date}` and
```

Comment:
there is no `${date}` placeholder in `DefaultFormatting`. should be `${time}` in comment